mne/channels/channels.py

```diff
@@ -1762,3 +1766,172 @@ def combine_channels(inst, groups, method='mean', keep_stim=False,
+                        ('Left-temporal', lt), ('Right-temporal', rt)])
+
+
+def _divide_side(lobe, x):
+    """Make a separation between left and right lobe evenly."""
+    lobe = np.asarray(lobe)
+    median = np.median(x[lobe])
+
+    left = lobe[np.where(x[lobe] < median)[0]]
+    right = lobe[np.where(x[lobe] > median)[0]]
+    medians = np.where(x[lobe] == median)[0]
+
+    left = np.sort(np.concatenate([left, lobe[medians[1::2]]]))
+    right = np.sort(np.concatenate([right, lobe[medians[::2]]]))
+    return list(left), list(right)
+
+
+@verbose
+def read_vectorview_selection(name, fname=None, info=None, verbose=None):
+    """Read Neuromag Vector View channel selection from a file.
+
+    By default, the selections used in ``mne_browse_raw`` are supported.
+    Additional selections can be added by specifying a selection file (e.g.
+    produced using ``mne_browse_raw``) using the ``fname`` parameter.
+
+    The ``name`` parameter can be a string or a list of string. The returned
+    selection will be the combination of all selections in the file where
+    (at least) one element in name is a substring of the selection name in
+    the file. For example, ``name=['temporal', 'Right-frontal']`` will produce
+    a combination of ``'Left-temporal'``, ``'Right-temporal'``, and
+    ``'Right-frontal'``.
+
+    Parameters
+    ----------
+    name : str | list of str
+        Name of the selection. If is a list, the selections are combined.
+        Supported selections are: ``'Vertex'``, ``'Left-temporal'``,
+        ``'Right-temporal'``, ``'Left-parietal'``, ``'Right-parietal'``,
+        ``'Left-occipital'``, ``'Right-occipital'``, ``'Left-frontal'`` and
+        ``'Right-frontal'``.
+    fname : str
+        Filename of the selection file (if None, built-in selections are used).
+    info : instance of Info
+        Measurement info file, which will be used to determine the spacing
+        of channel names to return, e.g. ``'MEG 0111'`` for old Neuromag
+        systems and ``'MEG0111'`` for new ones.
+    %(verbose)s
+
+    Returns
+    -------
+    sel : list of str
+        List with channel names in the selection.
+    """
+    # convert name to list of string
+    if not isinstance(name, (list, tuple)):
+        name = [name]
+    if isinstance(info, Info):
+        picks = pick_types(info, meg=True, exclude=())
+        if len(picks) > 0 and ' ' not in info['ch_names'][picks[0]]:
+            spacing = 'new'
+        else:
+            spacing = 'old'
+    elif info is not None:
+        raise TypeError('info must be an instance of Info or None, not %s'
+                        % (type(info),))
+    else:  # info is None
+        spacing = 'old'
+
+    # use built-in selections by default
+    if fname is None:
+        fname = op.join(op.dirname(__file__), '..', 'data', 'mne_analyze.sel')
+
+    if not op.isfile(fname):
+        raise ValueError('The file %s does not exist.' % fname)
```

Comment:
If you want to save coverage here you could use
```suggestion
    fname = _check_fname(fname, must_exist=True, overwrite='read')
```